### ISSUE TITLE
Multi Tagging the image won't push the image

### ISSUE THREAD

**Issue by stefc** [dotnet/sdk-container-builds#282]
```
   <ContainerImageTag>latest</ContainerImageTag>
    <!-- <ContainerImageTags>1.2.4-alpha2;latest</ContainerImageTags> -->
```

The single push the image correctly, put if I uncomment the second line and comment out the first on the following happens after publish : 

```
helloworld-dotnet7 dotnet publish --os linux --arch x64 -c Release -p:PublishProfile=DefaultContainer /t:PublishContainer 

MSBuild version 17.4.0+18d5aef85 for .NET
  Wiederherzustellende Projekte werden ermittelt...
  "/Users/stefc/Projects/stefc/google-cloud/helloworld-dotnet7/helloworld-dotnet7.csproj" wiederhergestellt (in "102 ms").
  helloworld-dotnet7 -> /Users/stefc/Projects/stefc/google-cloud/helloworld-dotnet7/bin/Release/net7.0/linux-x64/helloworld-dotnet7.dll
  helloworld-dotnet7 -> /Users/stefc/Projects/stefc/google-cloud/helloworld-dotnet7/bin/Release/net7.0/linux-x64/publish/
  Building image 'gcloud-run' with tags latest on top of base image mcr.microsoft.com/dotnet/runtime:7.0
  Pushed container 'gcloud-run:latest' to Docker daemon
➜  helloworld-dotnet7 dotnet publish --os linux --arch x64 -c Release -p:PublishProfile=DefaultContainer /t:PublishContainer

MSBuild version 17.4.0+18d5aef85 for .NET
  Wiederherzustellende Projekte werden ermittelt...
  Alle Projekte sind für die Wiederherstellung auf dem neuesten Stand.
  helloworld-dotnet7 -> /Users/stefc/Projects/stefc/google-cloud/helloworld-dotnet7/bin/Release/net7.0/linux-x64/helloworld-dotnet7.dll
  helloworld-dotnet7 -> /Users/stefc/Projects/stefc/google-cloud/helloworld-dotnet7/bin/Release/net7.0/linux-x64/publish/
  Building image 'gcloud-run' with tags  on top of base image mcr.microsoft.com/dotnet/runtime:7.0
➜  helloworld-dotnet7 

```

In the first call you see a "pushed container" message, this won't happen if I try to set two tags and neither the image appears

**Comment by baronfel** [comment]
Can you try with more recent nightly builds published to GitHub Packages? Rainer fixed this, we just haven't pushed out a release with the change yet. You can find instructions for using the nightly builds in DEVGUIDE.md.

**Comment by baronfel** [comment]
This should be fixed in version 0.3.2, released today. Give it a try and let us know!
